src/runtime/prte_globals.c

```diff
@@ -378,6 +378,51 @@ int prte_dt_init(void)
     return PRTE_SUCCESS;
 }
 
+void prte_display_prte_job_data(const char * msg)
+{
+    int rc;
+    uint32_t key;
+    prte_job_t *child_jdata=NULL, *jptr = NULL;
+    void *nptr;
+    int i = 0;
+
+    /* if the job data wasn't setup, we cannot provide the data */
+    if (NULL == prte_job_data) {
+        prte_output(0, "================================= %s (Not Initialized)", msg);
+        return;
+    }
+
+    prte_output(0, "================================= %s (BEGIN)", msg);
+
+    rc = prte_hash_table_get_first_key_uint32(prte_job_data, &key, (void **)&jptr, &nptr);
+    while (PRTE_SUCCESS == rc) {
+        if (NULL == jptr) {
+            continue;
+        }
+
+        prte_output(0, "%s (%d) %s at %s: Children %d",
```

Comment:
Maybe Children should be ChildCount or NumChildren

Reply:
fixed